src/views/UploadImportOrders.vue

```diff
@@ -141,7 +142,7 @@ export default defineComponent({
 
       const uploadData = this.content.map((order: any) => ({
         'orderIdValue': order[this.fieldMapping['orderId'].value],
-        'externalFacilityId': order[this.fieldMapping['facilityId'].value],
+        'externalFacilityId': order[this.fieldMapping['facilityId'].value] ? order[this.fieldMapping['facilityId'].value] : this.currentFacility?.facilityId,
```

Comment:
We can't pass facilityId here as the column is used to map using externalFacilityId

Reply:
We need to pass facilityId as a separate key, if mapping is not available.